packages/message-list/test/message-list-markdown-import.test.js

```diff
@@ -0,0 +1,39 @@
+import { expect } from '@vaadin/chai-plugins';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import '../src/vaadin-message-list.js';
+
+async function until(predicate) {
+  while (!predicate()) {
+    await new Promise((r) => {
+      setTimeout(r, 10);
+    });
+  }
+  return predicate();
+}
+
+describe('message-list-markdown dynamic import', () => {
+  let messageList;
+  const messages = [
+    {
+      text: 'This is a **bold text** in Markdown',
+      time: '10:00 AM',
+      userName: 'Markdown User',
+      userAbbr: 'MU',
+    },
+  ];
+
+  beforeEach(() => {
+    messageList = fixtureSync('<vaadin-message-list></vaadin-message-list>');
+  });
+
+  it('should not render incorrect content during import', async () => {
+    messageList.markdown = true;
+    messageList.items = messages;
+
+    // Expect the markdown to not exist in DOM as such
+    expect(messageList.textContent).to.not.include('**bold text**');
```

Comment:
Could also test that the respective message is hidden?

Reply:
I was considering it but it kind of feels like testing a technical detail. Having the messages hidden at any point isn't exactly an expected state of the component, but rather a workaround. Hiding the host would be as correct as hiding the individual messages but it would conflict with the test. If something, a visual test could make sense here but timing it might become flaky (and effectively it would just be a white box).

Reply:
OK, let's keep as is for now.